### PR TITLE
Added Python operators

### LINKED ISSUE #307: Add Python Logic/Math Operators
# Description

Logic/Math Operators for Python aren't presently on Code Thesaurus. This issue is to add that concept to the system.

# Requirements

* Copy `web/thesauruses/_meta/operators.json` to the right Python directory in `web/thesauruses/`
* Edit the operators.json file to accurately reflect the latest major version of Python
* Delete the `"name":` fields before your PR
* Add in `"comment":` if you wish to explain something in the code
* Add in `"not-implemented": "true"` if the concept doesn't exist in the language. You can then leave `"code":` blank.

# Claiming This Issue

Please leave a comment below so I can assign you to it. This is to prevent multiple people from working on the same issue. 

Check out [CONTRIBUTING.md](https://github.com/codethesaurus/codethesaur.us/blob/main/CONTRIBUTING.md) for more information on how to contribute. 

Check out [https://docs.codethesaur.us/thesaurus/](https://docs.codethesaur.us/thesaurus/) for information on how Code Thesaurus works and how to add to the thesauruses.

# Need Help?

Leave a comment on here with your question, or feel free to ask [@geekygirlsarah](https://twitter.com/geekygirlsarah) on Twitter. You can also email coreteam@codethesaur.us too.

# Thanks!

We really value and appreciate the additions people make to Code Thesaurus, no matter how large or small. So thank you for taking the time to work on this!

### PR DESCRIPTION
**Pull request by RiyaM16**
## What GitHub issue does this PR apply to?

Resolves #307 

## What changed and why?

Added Python operators.

## (If editing Django app) Please add screenshots

NA

## Checklist

- [X] I claimed any associated issue(s) and they are not someone else's
- [X] I have looked at documentation to ensure I made any revisions correctly
- [X] I tested my changes locally to ensure they work
- [ ] (If editing Django) I have added or edited any appropriate unit tests for my changes

## Any additional comments or things to be aware of while reviewing?

NA

### CODE REVIEW

File: web/thesauruses/python/operators.json [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+    },
+    "modulus": {
+      "code": "%"
+    },
+    "modulus_assignment": {
+      "code": "%="
+    },
+    "increment": {
+      "not-implemented": "true",
+      "code": ""
```
Comment (geekygirlsarah) [inline web/thesauruses/python/operators.json]: If there's concepts that are marked `not-implemented`, can you remove the `code` blocks? It'll make the files a little cleaner as they're not needed here.

**Review by geekygirlsarah — CHANGES_REQUESTED** [review]
I think if you make this small change, I can merge it in!

**Comment by RiyaM16** [comment]
Hi @geekygirlsarah!
I've made the needed changes.

**Comment by geekygirlsarah** [comment]
Yeah, I think this looks good to me! I'll merge it.

Thanks for working on this! Don't forget to sign up for Hacktoberfest if you haven't already, this will count towards it!

**Pull request merged**